doc/source/examples/incompressible-flow/3d-flow-around-sphere/flow-around-sphere.rst

```diff
@@ -0,0 +1,329 @@
+    subsection fluid 0
+      set kinematic viscosity = 10
+    end
+  end
+
+* In ``/examples/incompressible_flow/3d_flow_around_sphere/sphere_150.prm`` and ``/examples/incompressible_flow/3d_flow_around_sphere/sphere_adapt.prm`` (:math:`Re=150`)
+
+.. code-block:: text
+
+  subsection physical properties
+    subsection fluid 0
+      set kinematic viscosity = 0.006666667
+    end
+  end
+
+By default, simulations only contain a single fluid which is labeled ``0``.
+
+
+FEM interpolation
+~~~~~~~~~~~~~~~~~
+
+The default FEM parameters for this example use first order polynomials. They can be changed to Q2-Q1 elements.
+
+We specify the interpolation order for both pressure and velocity using the ``FEM`` subsection:
+
+.. code-block:: text
+
+    subsection FEM
+        set velocity order            = 1
+        set pressure order            = 1
+    end
+
+.. warning:: 
+
+    If you choose to use the ``gd_navier_stokes_3d`` solver; only Q2-Q1 elements are supported. 
+
+
+Simulation control
+~~~~~~~~~~~~~~~~~~
+
+The parameters also slightly differ from one problem to another as different examples experience different flow regimes.
+
+In fact, for the Re = 0.1, we have a laminar to creeping flow, meaning that the flow is very steady, and therefore can be solved in steady-state.
+
+.. code-block:: text
+
+  subsection simulation control
+    set method                  = steady
+    set number mesh adapt       = 0
+    set output name             = sphere-output
+    set output frequency        = 1
+    set subdivision             = 1
+  end
+
+At Re = 150, the flow has separated, resulting in an unstable wake and recirculation. It is hence more difficult to converge to a steady-state solution. 
+
+.. code-block:: text
+
+  subsection simulation control
+    set method                  = steady_bdf
+    set time step        	      = 0.1
+    set adapt 		              = true
+    set max cfl		              = 1000
+    set stop tolerance          = 1e-5
+    set adaptative time step scaling = 1.2
+    set number mesh adapt       = 0
+    set output name             = sphere-output
+    set output frequency        = 1
+    set subdivision             = 1
+  end
+
+The ``steady_bdf`` method solves for a steady-state simulation using adjoint time stepping with a bdf1 scheme. An initial time step is used to complete a transient iteration, and with each iteration, the time step is increased. The simulation is considered to have reached steady-state when the L2 norm of the initial residual is lower than stop tolerance at the start of a non-linear solution step, i.e. until the time step is large enough that a pseudo-steady-state has been reached.
+
+
+Initial condition
+~~~~~~~~~~~~~~~~~
+
+An initial condition is introduced for the Re = 150 problem. This way, a Re = 10 initial condition is set. Since the solution can easily be found at Re = 10, this is used as an initial attempt to hence find the solution at :math:`Re=150`.
```

Comment:
suggestion : this is used as an initial guess to find the solution at :math:`Re=150`.